AlphaGo/training/supervised_policy_trainer.py

```diff
@@ -85,6 +85,57 @@ def on_epoch_end(self, epoch, logs={}):
 }
 
 
+def load_indices_from_file(symmetries, shuffle_file):
+    # load indices from file
+    with open(shuffle_file, "r") as f:
+        indices = np.load(f)
+
+    return indices
+
+
+def set_indices(args, n_total_data_size, symmetries, shuffle_file_train, shuffle_file_val):
```

Comment:
Also, would it make sense for this function to always generate an array with all 8 symmetries? What happens if I (for some reason) start training with `--symmetries=none` but later resume training with `--symmetries=all`?